pkg/safeguards/safeguards.go

```diff
@@ -68,23 +64,40 @@ func GetManifestViolations(ctx context.Context, manifestFiles []string) ([]Manif
 		return manifestViolations, err
 	}
 
+	// organized map of manifest object by file name
+	manifestMap := make(map[string][]*unstructured.Unstructured, 0)
+
+	// aggregate of every manifest object into one list
+	objects := []*unstructured.Unstructured{}
+
 	for _, m := range manifestFiles {
-		var objectViolations map[string][]string
-		objects, err := fc.ReadManifests(m) // read all the objects stored in a single file
+		objs, err := fc.ReadManifests(m.Path) // read all the objects stored in a single file
 		if err != nil {
 			log.Errorf("reading objects %s", err.Error())
 			return manifestViolations, err
 		}
 
+		objects = append(objects, objs...)
+		manifestMap[m.Name] = objs
+	}
+
+	// thbarnes: loadData loads manifest data into client for review as well
+	if len(objects) > 0 {
+		err = loadData(ctx, c, objects)
```

Comment:
nit: renaming this method to something more precise would negate the need for a comment explaining where data is being loaded to/from

Reply:
fixed